source/index.txt

```diff
@@ -1,10 +1,141 @@
-===================
-Relational Migrator
-===================
+:template: product-landing
+:hidefeedback: header
+:noprevnext:
 
-.. default-domain:: mongodb
+===========================
+What is Relational Migrator
+===========================
 
-Your words here. Don't forget to add a toctree!
+.. introduction::
 
-Have a lovely day!
+   MongoDB Relational Migrator is a tool to help you migrate relational workloads to MongoDB. 
+   Relational Migrator allows you to:
 
+   - Design an effective MongoDB schema, derived from an existing relational schema.
+   - Migrate data from Oracle, SQL Server, MySQL, PostgreSQL to MongoDB, 
+     while transforming to the target schema.
+   - Generate code artifacts to reduce the time required to update application code.
+
+   |
+
+   .. button:: Download Relational Migrator
+      :uri: http://mongodb.com/try/download/relational-migrator
+
+.. image:: /img/hero.png
+   :alt: Homepage hero image
+
+.. kicker:: What You Can Do
+
+Bring Your Relational Workloads to MongoDB With Confidence
+----------------------------------------------------------
+
+.. tabs::
+
+   .. tab:: Mapping and Modelling
```

Comment:
Notably there is no links to the relevant docs in the tabs. This is because we are merging this to master which does not currently have access to any refs for the instruction pages.
I suggest that we add this as an item to the [Migrator Publish Plan Doc](https://docs.google.com/document/d/19_H1sWB878rYBVuyMirHRwQvRb_e5ynpr4WwG_QM2po/edit) and add the links after pre-local is merged into master.